File: base16/pygments/pygments_base16/base16-rose-pine-moon.py
from pygments.style import Style
from pygments.token import (
    Comment, Error, Keyword, Literal, Name, Number, Operator, String, Text
)


class Base16Style(Style):
    base00 = '#232136'
    base01 = '#2a273f'
    base02 = '#393552'
    base03 = '#59546d'
    base04 = '#817c9c'
    base05 = '#e0def4'
    base06 = '#f5f5f7'
    base07 = '#d9d7e1'
    base08 = '#ecebf0'
    base09 = '#eb6f92'
    base0a = '#f6c177'
    base0b = '#ea9a97'
    base0c = '#3e8fb0'
    base0d = '#9ccfd8'
    base0e = '#c4a7e7'
    base0f = '#b9b9bc'

    default_style = ''

    background_color = base00
    highlight_color = base02

    styles = {
        Text: base05,
        Error: base08,  # .err

        Comment: base03,  # .c
        Comment.Preproc: base0f,  # .cp
        Comment.PreprocFile: base0b,  # .cpf

        Keyword: base0e,  # .k
        Keyword.Type: base08,  # .kt

        Name.Attribute: base0d,  # .na
        Name.Builtin: base0d,  # .nb
        Name.Builtin.Pseudo: base08,  # .bp
        Name.Class: base0d,  # .nc
        Name.Constant: base09,  # .no
        Name.Decorator: base09,  # .nd
        Name.Function: base0d,  # .nf
        Name.Namespace: base0d,  # .nn
        Name.Tag: base0e,  # .nt
        Name.Variable: base0d,  # .nv
        Name.Variable.Instance: base08,  # .vi

        Number: base09,  # .m

        Operator: base0c,  # .o
        Operator.Word: base0e,  # .ow

        Literal: base0b,  # .l

        String: base0b,  # .s
        String.Interpol: base0f,  # .si
        String.Regex: base0c,  # .sr
        String.Symbol: base09,  # .ss
    }


from string import capwords  # noqa: E402
Base16Style.__name__ = 'Base16{}Style'.format(
    capwords('rose-pine-moon', '-').replace('-', '')
)
globals()[Base16Style.__name__] = globals()['Base16Style']
del globals()['Base16Style']
del capwords

File: base16/crosh/colors/base16-rose-pine-moon.js
// Base16 Rosé Pine Moon
// Scheme: Emilia Dunfelt &lt;edun@dunfelt.se&gt;

var color_scheme = {
        'base00': '#232136',
        'base01': '#2a273f',
        'base02': '#393552',
        'base03': '#6e6a86',
        'base04': '#908caa',
        'base05': '#e0def4',
        'base06': '#e0def4',
        'base07': '#56526e',
        'base08': '#eb6f92',
        'base09': '#f6c177',
        'base0A': '#ea9a97',
        'base0B': '#3e8fb0',
        'base0C': '#9ccfd8',
        'base0D': '#c4a7e7',
        'base0E': '#f6c177',
        'base0F': '#56526e',
};

term_.prefs_.set('background-color', color_scheme.base00);
term_.prefs_.set('foreground-color', color_scheme.base05);
term_.prefs_.set('cursor-color', "rgba(224, 222, 244, 0.5)");

term_.prefs_.set('color-palette-overrides', 
                        [color_scheme.base00,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base05,
                        color_scheme.base03,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base07,
                        color_scheme.base09,
                        color_scheme.base0F,
                        color_scheme.base01,
                        color_scheme.base02,
                        color_scheme.base04,
                        color_scheme.base06]);
